api-references/aquarius-rest-api.md

```diff
@@ -78,7 +78,7 @@
     curl --location --request POST 'https://v4.aquarius.oceanprotocol.com/api/aquarius/assets/names' \
```

Comment:
I got the response below, let me know if I'm doing anything wrong?

```
curl --location --request POST 'https://v4.aquarius.oceanprotocol.com/api/aquarius/assets/names' --header 'Content-Type: application/json'  --data-raw '{"didList" : ["did:op:CbD7aeecB5DFbABaB9126B5Cf1262dCFBA178479"] "didList" : ["did:op:cd086344c275bc7c560e91d472be069a24921e73a2c3798fb2b8caadf8d245d6"]}'

<!doctype html>
<html lang=en>
<title>400 Bad Request</title>
<h1>Bad Request</h1>
<p>The browser (or proxy) sent a request that this server could not understand.</p>

```

Reply:
Your DID has capital letters, I tested with `did:op:cd086344c275bc7c560e91d472be069a24921e73a2c3798fb2b8caadf8d245d6` and the console output response was 200 OK.

Reply:
Yeah works fine. Thanks a lot